code/modules/paperwork/photocopier.dm

```diff
@@ -151,11 +169,12 @@
 				else if(toner >= 5 && !busy && check_ass()) //You have to be sitting on the copier and either be a xeno or a human without clothes on.
 					if(isalienadult(ass) || istype(ass, /mob/living/simple_animal/hostile/alien)) //Xenos have their own asses, thanks to Pybro.
 						temp_img = "alien"
-					else if(isdrone(ass))
+					else if(isdrone(ass)) //Drones are hot

```

Comment:
ok listen here

Reply:
For some reason this was removed. I re-added it.